yt_dlp/extractor/arte.py

```diff
@@ -110,6 +111,13 @@ def _real_extract(self, url):
             countries = tuple(code for code in geoblocking['code'].split('_') if code != 'EUR')
             raise GeoRestrictedError('Video is geoblocked', countries=countries)
 
+        # e.g. <https://www.arte.tv/es/videos/104351-002-A/serviteur-du-peuple-1-23/>
+        # (only available in the /fr/ and /de/ versions)
+        # videos that are completely nonexistent return HTTP 404
+
+        if config['data']['attributes']['streams'] is None:
```

Comment:
This should probably check `rights` instead. <https://www.arte.tv/de/videos/097407-215-A/28-minuten/> returns `rights: null` and `streams: []`.